### ISSUE TITLE
Doesn't dox2html5.py support 'retval' keyword?

### ISSUE THREAD

**Issue by toolbits** [mosra/m.css#27]
Hi, I'm trying to use dox2html5.py to make a document for my project.
However, it seems not supporting retval keyword in current version.

I had tried to customize dox2html5.py script and templates to support retval, but, I couldn't success it.

Do you have some plan to support retval?

**Comment by mosra** [comment]
Hi! Isn't `@retval` just an alias to `@return`? The `@return` command is definitely supported, here's how it looks in the output, for example: http://doc.magnum.graphics/magnum/namespaceMagnum_1_1MeshTools.html#a425eb14ea8166f1648bd0aef57010231

**Comment by toolbits** [comment]
Thank you for your quick reply!!
@retval and @return are not same thing.
I know that dox2html5.py supports @return keyword and definitely generates correct output.
@retval keyword generates a list of return values.

Please see my project's document (but, currently written in Japanese, sorry...)
http://toolbits.github.io/GPDSP/d1/d09/classir_1_1_g_p_d_s_p_buffer_input_node.html
You can find difference between them at setPosition() and getPosition() descriptions.

**Comment by mosra** [comment]
I see the difference now, thank you. I can implement it, yes, though I won't be able to get to that until Friday or so. Hope that's okay :)

**Comment by toolbits** [comment]
Wow, that is good news.
I'll wait your implementation of `@retval`. Thank you.

some infos: `@retval` style vs `@return` style

```
@retval <label1> <description1 (omittable)>
@retval <label2> <description2 (omittable)>
@retval <label3> <description3 (omittable)>
@retval <label4>

@return <description>

```

**Comment by mosra** [comment]
Support for the `@retval` command is implemented in 5e6d0bcaae00412c3acbd371f11ef6276af6d0b9. Please reopen if you find any issues with it :)
